### PR TITLE
bears/js: Add HappinessLintBear

### PR DESCRIPTION
**Pull request by mr-karan**
This bear uses `happiness` to lint
`.js` files.

Closes https://github.com/coala-analyzer/coala-bears/issues/588

### CODE REVIEW

File: tests/js/HappinessLintBearTest.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+console.log(value)
+run(function (err) {
+  window.alert('done')
+})
+if (options.quiet !== true)
+  console.log('done')
+var x=2
+var message = 'hello, '+name+'!'
+var list = [1,2,3,4]
+""".splitlines(keepends=True)
```
Comment (Asnelchristian) [inline tests/js/HappinessLintBearTest.py]: is it possible to shorten the test files :)

Comment (mr-karan) [inline tests/js/HappinessLintBearTest.py]: done :) :+1:

**Comment by Adrianzatreanu** [comment]
you got @arafsheikh 's commit for some reason :D

**Comment by mr-karan** [comment]
shifted to https://github.com/coala-analyzer/coala-bears/pull/592, sorry.

**Pull request closed without merging**